pkg/controller/apis/config/types.go

```diff
@@ -89,6 +89,8 @@ type KubeControllerManagerConfiguration struct {
 	HPAController poautosclerconfig.HPAControllerConfiguration
 	// JobControllerConfiguration holds configuration for JobController related features.
 	JobController jobconfig.JobControllerConfiguration
+	// CronJobControllerConfiguration holds configuration for CronJobController related features.
+	CronJobController jobconfig.CronJobControllerConfiguration
```

Comment:
Since this applies to the new controller use `CronJobControllerV2`, instead.

Reply:
done